presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -511,6 +511,14 @@ private static boolean isPrestoQueryInvalid(SQLException e)
 
     private static boolean resultsMatch(QueryResult controlResult, QueryResult testResult, int precision)
     {
+        if (controlResult.getResults() == null && testResult.getResults() == null) {
```

Comment:
Instead of adding this check, how about changing `Validator.executeQuery` to call `getUpdateCount` for non-SELECT queries?